src/lib/utils.c

```diff
@@ -131,8 +131,13 @@ int get_int(const char *path, int defval, const char *name)
 	return defval;
 }
 
-/**
- */
+bool is_subpath(const char * const path, const char * const subpath, size_t subpath_strlen)
+{
+	if (strncmp(path, subpath, subpath_strlen) == 0)
```

Comment:
I think it's worth adding a check whether path and subpath are null,
in the case when strncmp gets NULL it will result in a segfault

Reply:
We should not hide programmer mistakes by wrapping return result. If Null is passed then it is a developer mistake and it is easy to catch. I'm against adding dead code like check you proposed. I will add assert.